server/src/db/migrations/1_initial-migrations.sql

```diff
@@ -1,6 +1,15 @@
-CREATE TABLE student (
-  student_id SERIAL PRIMARY KEY,
+CREATE EXTENSION "uuid-ossp";
+
+CREATE TABLE "user" (
+  user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
-  birth_year INT NOT NULL
+  roles TEXT[] NOT NULL CHECK (roles <@ '{"para", "case_manager", "admin"}')
+);
+
+CREATE TABLE "student" (
+  student_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  first_name TEXT NOT NULL,
+  last_name TEXT NOT NULL,
+  assigned_case_manager_id UUID REFERENCES "user" (user_id) ON DELETE SET NULL
```

Comment:
Is this a property of the student of the IEP? as in can a student be working with multiple case managers?

Reply:
Good question, from what I've seen so far it sounds like students are assigned a case manager but it's worth clarifying.

Reply:
I think the direction of case_manager -> student link is more important, than student -> case_manager (i.e. we should probably have the student property in the users table).

Reply:
One case manager has many students though, right?

Reply:
Yes, a CM will have multiple students. One of the first things we will have to build is the CM dashboard, where they can  add/remove students and edit their IEPs.